optuna/study/study.py

```diff
@@ -489,9 +489,9 @@ def tell(
         self,
         trial: Union[trial_module.Trial, int],
         values: Optional[Union[float, Sequence[float]]] = None,
-        state: TrialState = TrialState.COMPLETE,
+        state: Optional[TrialState] = None,
         skip_if_finished: bool = False,
-    ) -> None:
+    ) -> FrozenTrial:
```

Comment:
👍

Reply:
By the way, how about updating the docs, about the return type, `state` parameter and the new failing behavior?

Reply:
9ef0d75 and 2cc402b update the documentation. ☝️